schemas/2014-10-01-preview/microsoft.authorization.json

```diff
@@ -3,7 +3,7 @@
     "$schema": "http://json-schema.org/draft-04/schema#",
     "title": "Microsoft.Authorization",
     "description": "Microsoft Microsoft.Authorization Resource Types",
-    "definitions": {
+    "resourceDefinitions": {
```

Comment:
"httpActionRequest" in this file is not a resource, we should keep it as "definitions"